GlobalConfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We should add paths for where software should be downloaded to locally and where applications files should be stored for SCCM.

I think it would also make sense to have the SCCM be an array much like the suggestion I have for recipeLocations so we can have multiple targets.

Reply:
Added additional information for these to the ConfigMgr array.